pkg/plugins/handlers.go

```diff
@@ -17,32 +17,73 @@ import (
 type PluginsHandler struct {
 	Client             *http.Client
 	PluginsEndpointMap map[string]string
+	PublicDir          string
 }
 
-func NewPluginsHandler(client *http.Client, token string, pluginsEndpointMap map[string]string) *PluginsHandler {
+func NewPluginsHandler(client *http.Client, pluginsEndpointMap map[string]string, publicDir string) *PluginsHandler {
 	return &PluginsHandler{
 		Client:             client,
 		PluginsEndpointMap: pluginsEndpointMap,
+		PublicDir:          publicDir,
 	}
 }
 
-func (p *PluginsHandler) HandlePlugins(w http.ResponseWriter, r *http.Request) {
+func (p *PluginsHandler) HandlePluginLocales(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		w.Header().Set("Allow", "GET")
+		serverutils.SendResponse(w, http.StatusMethodNotAllowed, serverutils.ApiError{Err: "Method unsupported, the only supported methods is GET"})
+		return
+	}
+
+	query := r.URL.Query()
+	lang := query.Get("lng")
+	// In case of static plugin the localization namespace should contain name of the json file that contains the localization strings,
```

Comment:
```suggestion
	// In case of console app or static plugins, the localization namespace should contain name of the json file that contains the localized strings,
```